### PR TITLE
Remove projects no longer using Docsify

### PR DESCRIPTION
**Pull request by trusktr**
Removes outdated projects from the showcase (f.e. they no longer use Docsify or no longer exist).

I also moved some items to a separate "Community Resources" section, where I think those projects will have better exposure for their purpose.

### CODE REVIEW

**Comment by jthegedus** [comment]
Is this still WIP?

**Comment by trusktr** [comment]
@jthegedus  Not anymore, just updated it.

**Comment by trusktr** [comment]
For someone who is exploring Docsify, and may randomly click on some of the showcase items, they may not be too impressed. Some of the examples aren't too great. Maybe there should be a "staff picks" section to highlight the best Docsify sites, so that people who are exploring can get the best inspiration from them.

**Comment by jthegedus** [comment]
> Maybe there should be a "staff picks" section to highlight the best 
> Maybe we can have a community vote (or something) for top Docsify sites

@trusktr That is a very good idea

**Comment by trusktr** [comment]
I also moved some items to a separate "Community Resources" section, where I think those projects will have better exposure for their purpose.

**Comment by jthegedus** [comment]
@trusktr Are you able to address the lint issues?

**Comment by trusktr** [comment]
Let me see about those. I gotta get off for now though but I can come back later. I don't mind if you commit here too though.

@jthegedus  Are you on Docsify Discord yet? I didn't see your username there (or is it different?). Invitation link: https://discord.gg/3NwKFyR

**Comment by jthegedus** [comment]
I didn't know we had a Discord 😅 Unfortunately I have been rather inactive on this project which I hope to fix soon.

File: README.md [hunk 1]
```diff
@@ -122,26 +117,24 @@ A curated list of awesome things related to <a href="//docsify.js.org">docsify</
...
 - [🌙 docsify-darkly-theme](https://github.com/sushantrahate/docsify-darkly-theme) - Dark theme for docsify. [@sushantrahate](https://github.com/sushantrahate).
-- [:first_quarter_moon:docsify-darklight-theme](https://github.com/boopathikumar018/docsify-darklight-theme) - It is a simple and highly customizable theme for docsify.js. Using this theme, documents will have a theme switcher to switch between  `dark-mode` and  `light-mode`. See [preview](https://docsify-darklight-theme.boopathikumar.me) here. [@boopathikumar](https://boopathikumar.me/).
-
+- [:first_quarter_moon:docsify-darklight-theme](https://github.com/boopathikumar018/docsify-darklight-theme) - It is a simple and highly customizable theme for docsify.js. Using this theme, documents will have a theme switcher to switch between `dark-mode` and `light-mode`. See [preview](https://docsify-darklight-theme.boopathikumar.me) here. [@boopathikumar](https://boopathikumar.me/).
 
 ## Enterprise Usage
 
 - Eleme - [vue-amap](https://elemefe.github.io/vue-amap/#/)
-- Alibaba - [weex-ui](https://alibaba.github.io/weex-ui/#/)
```
Comment (Koooooo-7) [inline README.md:144]: it seems they redirected the address [https://apache.github.io/incubator-weex-ui/#/](https://apache.github.io/incubator-weex-ui/#/) .

**Comment by anikethsaha** [comment]
> > Maybe there should be a "staff picks" section to highlight the best
> > Maybe we can have a community vote (or something) for top Docsify sites
> 
> @trusktr That is a very good idea
> 
> @anikethsaha What are your thoughts?

Yup, agreed this is a good idea. But I do think there are kind of less interaction from the community in this repo. 

But I like the idea

**Comment by jthegedus** [comment]
I'll resolve the conflicts here momentarily

**Comment by jthegedus** [comment]
@Koooooo-7 This should be good to go now, please re-review. Up to date with `master` and passes all CI check 💯 Thanks @trusktr

**Pull request merged**